lib/Runtime/PlatformAgnostic/Platform/Common/Intl.cpp

```diff
@@ -317,7 +339,7 @@ namespace Intl
             [&currencyDisplay, currencyCode](icu::Locale &locale, UErrorCode &error) -> icu::NumberFormat*
             {
                 icu::NumberFormat *nf = nullptr;
-                if (currencyDisplay == NumberFormatCurrencyDisplay::SYMBOL || currencyDisplay >= NumberFormatCurrencyDisplay::MAX)
```

Comment:
> currencyDisplay >= NumberFormatCurrencyDisplay::MAX [](start = 77, length = 52)

This was meant to catch the case that currencyDisplay was somehow initialized to some bogus value. I think you need to handle that case or failfast if it occurs. (If you prefer the latter add else { AssertOrFailFastMsg(...) }

Reply:
I made it equals because of ASSERT_ENUM. I can change that macro to do something in release mode separately.